sensor_msgs/LaserScan laserScan
float64 stepAngle